test/jdk/java/awt/Dialog/FileDialogIconTest/FileDialogIconTest.java

```diff
@@ -38,7 +39,7 @@
 
 /*
  * @test
- * @bug 4035189
+ * @bug 6425126
  * @summary Test to verify that PIT File Dialog icon not matching with
  *          the new java icon (frame Icon) - PIT build
  * @requires (os.family == "windows")
```

Comment:
Bug description says -- `This is reproduce on solaris & cinnabar. This works fine in Mustang b83.`

Why the test is restricted for `windows` only ?